packages/core/src/utils/load-plugins.ts

```diff
@@ -0,0 +1,43 @@
+import * as path from 'path';
+import Auto from '../auto';
+import { ILogger } from './logger';
+import tryRequire from './try-require';
+
+export type IPluginConstructor = new (options?: any) => IPlugin;
+
+export interface IPlugin {
+  name: string;
+  apply(auto: Auto): void;
+}
+
+export default function loadPlugin(
+  [pluginPath, options]: [string, any],
+  logger: ILogger
+): IPlugin | undefined {
+  let plugin = tryRequire(pluginPath) as (
+    | IPluginConstructor
+    | { default: IPluginConstructor });
+
+  if (!plugin) {
+    plugin = tryRequire(
+      path.join(process.cwd(), pluginPath)
+    ) as IPluginConstructor;
+  }
+
+  if (!plugin) {
+    plugin = tryRequire(
+      path.join('@intuit-auto', pluginPath)
```

Comment:
Instead of hardcoding something `intuit` specific here, could there be room to provide a config option like `defaultPluginScope` or something?

Reply:
Ohhh. Nevermind, I see. That's the scope you've used to deploy the default plugins on npm. Gotcha.